### PR TITLE
Challenge

### PR DESCRIPTION
**Pull request by ellisthomas**
@zoeames @morecallan @rousell

### CODE REVIEW

**Comment by ellisthomas** [comment]
Done!!

**Comment by morecallan** [comment]
Duuuude! This is looking great. I love the color scheme. Digging the layout. Also, your commit history is absolutely gorgeous. I have a few comments before you merge, though.

-Look over your indentation, I noticed a few places where it's a bit off.
-Looks like none of the images you referenced got pushed up to your branch
-Take this project spec literally: "Article element with two sections for future projects (placeholders)"
-Comment your code :) The project specs are a good place to start. 

Way to go with the first "Milestone Challenge"!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+		<article class="bio">
+			<h2>Duis ornare</h2>
+			<img class="profile-pic" src="/images/kid.png" alt="eliis">
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi pellentesque sapien vel ante eleifend pharetra. Vestibulum nec diam elementum, molestie enim ut, tristique lacus. Vivamus est ligula, posuere ut tristique non, ullamcorper vitae massa. Duis ornare, mauris non maximus fringilla, purus ipsum pretium metus, ut vehicula purus enim at nulla. Maecenas tempus elit ac quam pulvinar ornare. Proin non nibh semper, convallis dui ac, auctor purus. Fusce eu ullamcorper dolor. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus venenatis, velit non vehicula semper, est leo semper justo, sed tempus lorem augue ac lacus. Integer pellentesque est arcu, ultricies dictum dolor fermentum eget. Cras pellentesque erat sed pretium vehicula. Aliquam erat volutpat. Pellentesque ut tempor ex. Donec sodales tempor dolor a vestibulum.</p>
+		</article>
+
+		<article>
+			<h3>Links to resources or sites you like</h3>
+			 	<ol>
+			 		<li><a href="http://nashvillesoftwareschool.com/">Nashville Software School</li></a>
```
Comment (morecallan) [inline index.html]: Close out a tag before li tag here.

Comment (ellisthomas) [reply index.html]: Fixing indentions now and also im not seeing the tag I need to close

**Comment by ellisthomas** [comment]
Changes made and images uploaded! Thank you

**Comment by rousell** [comment]
👍 - good job!

**Comment by rousell** [comment]
hahah sorry for closing your PR! I meant to just merge it in :)

**Pull request merged**